python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -300,7 +300,8 @@ class NearestNeighbors(Base,
                  p=2,
                  algo_params=None,
                  metric_params=None,
-                 output_type=None):
+                 output_type=None,
+                 n_jobs=None):
```

Comment:
```suggestion
                **kwargs):
```

Since we don't actually use the parameter, it would be better to add kwargs to avoid  increasing the function signature, same thing is already done in RF: https://github.com/rapidsai/cuml/blob/903d5765abe40d0d29a75c470a36a9c2c1f8e927/python/cuml/ensemble/randomforestregressor.pyx#L264

Reply:
Thanks for this suggestion, Dante!  That makes a lot of sense!